### PR TITLE
[BUGFIX] use new constants in TYPO3 8.x

### PR DESCRIPTION
**Pull request by anjeylink**
Fix breaking change in for TYPO3 8.x
https://docs.typo3.org/typo3cms/extensions/core/8.7/Changelog/8.0/Breaking-72368-TYPO3ConstantsRemoved.html

### CODE REVIEW

**Comment by kaystrobach** [comment]
Thank you, will check lateron and make an 7.x branch finally.

File: Classes/Service/DyncssService.php [hunk 1]
```diff
@@ -103,4 +103,18 @@ public static function getOverrides()
...
+
+    /**
+     * Check CLI mode depending on TYPO3 version
+     *
+     * @return int
+     */
+    protected static function isCliMode()
+    {
+        if (version_compare(TYPO3_version, '8.0', '<')) {
+            return TYPO3_cliMode;
```
Comment (pniederlag) [inline Classes/Service/DyncssService.php]: I'd assume this will still trigger https://github.com/kaystrobach/TYPO3.dyncss/issues/54

**Comment by pniederlag** [comment]
https://github.com/kaystrobach/TYPO3.dyncss/pull/55 is fixing this imho in a better way

**Comment by kaystrobach** [comment]
see https://github.com/kaystrobach/TYPO3.dyncss/releases/tag/0.8.5

**Pull request merged**